controllers/common/controller.go

```diff
@@ -27,18 +27,14 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
How about move this controller into a standalone directory? (as one of the subdirectory of `common`).

Though it was named `common`, it was a standalone controller "records" like anyone else ("condition", "desiredphase"...)

Reply:
And the `README.md` should also be moved.

Reply:
Fixed